shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-opengauss/src/main/java/org/apache/shardingsphere/data/pipeline/opengauss/ingest/wal/decode/MppdbDecodingPlugin.java

```diff
@@ -138,6 +139,9 @@ private List<Object> getDeleteColumnDataFromMppDataEvent(final MppTableData data
     }
     
     private Object readColumnData(final String data, final String columnType) {
+        if (Objects.equals(data, "null")) {
```

Comment:
Why not `"null".equals(data)`?

Reply:
Just getting used to using this method, do I need to change it?